.github/workflows/kubetail-rpm.spec

```diff
@@ -0,0 +1,23 @@
+Name:           kubetail 
+Version:        %{version} 
+Release:        1%{?dist}
+Summary:        Go based cli to get logs from kubernetes 
+
+License:        Apache License Version 2.0
+URL:            https://github.com/kubetail-org/kubetail
+Source0:        %{name}-%{version}.tar.gz
+
+BuildArch:      %{arch} 
+
+%description
+A longer description of your Go CLI tool and what it does.
```

Comment:
Kubetail is a general-purpose logging dashboard for Kubernetes, optimized for tailing logs across across multi-container workloads in real-time. With Kubetail, you can view logs from all the containers in a workload (e.g. Deployment or DaemonSet) merged into a single, chronological timeline, delivered to your browser or terminal.